src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const dayPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
```

Comment:
Use constants that explain the values used in the expressions to avoid hardcoding. For example, define constants for the minimum number of days required to receive each discount, such as `SHORT_TERM_RENTAL_DAYS` and `LONG_TERM_RENTAL_DAYS`.